src/app/productManagement/pricing/templates/updateProductPrice.modal.html

```diff
@@ -1,4 +1,4 @@
-<form name="updateProductPriceModal.form">
+<form name="updateProductPriceModal.form" ng-submit="updateProductPriceModal.submit()" oc-pretty-submit>
```

Comment:
That works as expected, yes.

I noticed an issue when all the user does is delete a price break. It did not recognize the model as being updated.